src/web/app/components/session-edit-form/session-edit-form.component.html

```diff
@@ -132,7 +132,15 @@ <h5 class="text-md-right">Or: </h5>
             <div class="row text-center align-items-center">
               <div class="col-md-7 col-xs-center">
                 <div class="input-group">
-                  <input id="submission-start-date" type="text" class="form-control" ngbDatepicker [ngModel]="model.submissionStartDate" (ngModelChange)="triggerModelChange('submissionStartDate', $event)"  #startTimeDp="ngbDatepicker" [disabled]="!model.isEditable"/>
+                  <ng-template #footer1>
```

Comment:
Could we maybe try having more descriptive template names instead of just `footer1`? Maybe `submissionStartFooter` or something of the sort. Similarly for `footer2`, `footer3`, and `footer4`